src/panels/logbook/ha-panel-logbook.ts

```diff
@@ -147,13 +147,13 @@ export class HaPanelLogbook extends LitElement {
     todayEnd.setDate(todayEnd.getDate() + 1);
     todayEnd.setMilliseconds(todayEnd.getMilliseconds() - 1);
 
-    const todayCopy = new Date(today);
-
+    let todayCopy = new Date(today);
     const yesterday = new Date(todayCopy.setDate(today.getDate() - 1));
```

Comment:
I think the real issue would be this line? We should not set the date of `todayCopy`?

Reply:
Streamlined the coding to get rid of the `todayCopy`.